Libraries/Components/Pressable/Pressable.js

```diff
@@ -26,7 +26,8 @@ import {PressabilityDebugView} from '../../Pressability/PressabilityDebug';
 import usePressability from '../../Pressability/usePressability';
 import {normalizeRect, type RectOrSize} from '../../StyleSheet/Rect';
 import type {ColorValue} from '../../StyleSheet/StyleSheetTypes';
-import type {LayoutEvent, PressEvent} from '../../Types/CoreEventTypes';
+import type {LayoutEvent, MouseEvent, PressEvent} from '../../Types/CoreEventTypes'; // TODO(macOS ISS#2323203)
```

Comment:
It looks like the last CI error is just a lint error: it wants the items in the `{}` to be on their own lines.   Just run `yarn lint --fix`.

Reply:
Rebased and fixed my silly lint error.